code/modules/mob/living/basic/space_fauna/eyeball/_eyeball.dm

```diff
@@ -0,0 +1,118 @@
+
+	maxHealth = 30
+	health = 30
+	obj_damage = 10
+	melee_damage_lower = 8
+	melee_damage_upper = 12
+
+	attack_verb_continuous = "bites"
+	attack_verb_simple = "bite"
+	attack_sound = 'sound/weapons/bite.ogg'
+	attack_vis_effect = ATTACK_EFFECT_BITE
+
+	faction = list(FACTION_SPOOKY)
+	speak_emote = list("telepathically cries")
+
+	habitable_atmos = list("min_oxy" = 0, "max_oxy" = 0, "min_plas" = 0, "max_plas" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	minimum_survivable_temperature = T0C
+	maximum_survivable_temperature = T0C + 1500
+	sight = SEE_SELF|SEE_MOBS|SEE_OBJS|SEE_TURFS
+
+	lighting_cutoff_red = 40
+	lighting_cutoff_green = 20
+	lighting_cutoff_blue = 30
+
+	ai_controller = /datum/ai_controller/basic_controller/eyeball
+	///how much we will heal eyes
+	var/healing_factor = 3
+	/// is this eyeball crying?
+	var/crying = FALSE
+	/// the glare ability
+	var/datum/action/cooldown/spell/pointed/death_glare/glare
+	/// the crying overlay we add when is hit
+	var/mutable_appearance/on_hit_overlay
+
+	///cooldown to heal eyes
+	COOLDOWN_DECLARE(eye_healing)
+
+/mob/living/basic/eyeball/Initialize(mapload)
+	. = ..()
+	glare = new
+	glare.Grant(src)
+	ai_controller.set_blackboard_key(BB_GLARE_ABILITY, glare)
+	AddElement(/datum/element/simple_flying)
+	ADD_TRAIT(src, TRAIT_SPACEWALK, INNATE_TRAIT)
+	RegisterSignal(src, COMSIG_HOSTILE_PRE_ATTACKINGTARGET, PROC_REF(pre_attack))
+	on_hit_overlay = mutable_appearance(icon, "[icon_state]_crying")
+
+/mob/living/basic/eyeball/UnarmedAttack(atom/attack_target, proximity_flag, list/modifiers)
+	. = ..()
+	if(!.)
+		return
+
+	if(!proximity_flag)
+		return
+
+	if(istype(attack_target, /obj/item/food/grown/carrot))
+		adjustBruteLoss(-5)
+		to_chat(src, span_warning("You eat the [attack_target]! It restores some health"))
+		qdel(attack_target)
+		return TRUE
+
+/mob/living/basic/eyeball/attackby(obj/item/weapon, mob/living/carbon/human/user, list/modifiers)
+	. = ..()
+	if(!weapon.force && !user.combat_mode)
+		return
+	if(crying)
+		return
+	change_crying_state()
+	addtimer(CALLBACK(src, PROC_REF(change_crying_state)), 10 SECONDS) //cry for 10 seconds then remove
+
+/mob/living/basic/eyeball/proc/change_crying_state()
+	crying = !crying
+	if(crying)
+		add_overlay(on_hit_overlay)
+		return
+	cut_overlay(on_hit_overlay)
+
+
+/mob/living/basic/eyeball/proc/pre_attack(mob/living/eyeball, atom/target)
+	SIGNAL_HANDLER
+
+	if(!ishuman(target))
+		return
+
+	var/mob/living/carbon/human_target = target
+	var/obj/item/organ/internal/eyes/eyes = human_target.get_organ_slot(ORGAN_SLOT_EYES)
+	if(!eyes)
+		return
+	if(eyes.damage < 10)
+		return
+	heal_eye_damage(human_target, eyes)
+	return COMPONENT_HOSTILE_NO_ATTACK
+
+
+/mob/living/basic/eyeball/proc/heal_eye_damage(mob/living/target, obj/item/organ/internal/eyes/eyes)
+	if(!COOLDOWN_FINISHED(src, eye_healing))
+		return
+	to_chat(target, span_warning("[src] seems to be healing our [eyes]"))
```

Comment:
```suggestion
	to_chat(target, span_warning("[src] seems to be healing your [eyes.zone]."))
```
If we use zone it just says "healing your eyes" rather than "healing your the alien eyes".